### DISCUSSION TITLE
Support human friendly color palette?

### DISCUSSION THREAD

**Discussion by kdheepak** [ratatui/ratatui#1324] [Ideas]
We already have `tailwind` and `material`. There are two more color palettes that may be useful to support:

1. https://xkcd.com/color/rgb/
2. https://en.wikipedia.org/wiki/X11_color_names?useskin=vector#Color_name_chart

**Comment by joshka** [comment]
I think it would be nice to also support theming more generally, and then map the design of palettes into that idea as a component rather than a concept divorced from the themes.

**Comment by orhun** [comment]
We can create a good first issue for this :)
